icechunk/src/ops/gc.rs

```diff
@@ -286,6 +305,7 @@ async fn gc_chunks(
 
 #[instrument(skip(storage, storage_settings, config, keep_ids), fields(keep_ids.len = keep_ids.len()))]
 async fn gc_manifests(
+    asset_manager: &Arc<AssetManager>,
```

Comment:
do you need these to be Arcs? Or can they be &?

Reply:
Looks like i can do `&`.